### ISSUE TITLE
Error: Request entity too large

### ISSUE THREAD

**Issue by KalachevDev** [embermap/ember-cli-fastboot-testing#78]
I'm using ember-cli-fastboot-testing in couple with ember-cli-mirage. Sometimes returned data is very large (returned by factories) and as a result ExpressJS throws and error: `Request entity too large`. I was able to fix it by adding following lines (by specifying limit in (specify a limit in bodyparser). I can create PR with this changes, but probably we might allow user to configure some other parameters in bodyParser and implement this with external config (possible might lead to more issues since we will give more complex API to addon user)?

`app.post('/__mock-request', bodyParser.json({ limit: '50mb' }), (req, res) => {`

**Comment by CvX** [comment]
Raising the limit sounds good! The default 100kb is definitely too low for many reasonable cases. 🙂

I wouldn’t expose bodyParser options to ECFT users. It’s an implementation details that could change (e.g. if we move communication from http to web sockets).

**Comment by KalachevDev** [comment]
@CvX Agreed! Would you like me to raise PR with these changes?

**Comment by CvX** [comment]
Yeah, that would be great! 😃

**Comment by ryanto** [comment]
Yup, totally agree!

**Comment by KalachevDev** [comment]
Opened https://github.com/embermap/ember-cli-fastboot-testing/pull/79

**Comment by ryanto** [comment]
Hey @KalachevDev you mentioned you're using ember-cli-fastboot-testing with ember-cli-mirage. I'd love to hear about how you've set that up?

**Comment by CvX** [comment]
I’m also using mirage factories with ECFT. It’s something along those lines:

```js
import { module, test } from 'qunit';
import { setup, visit, mockServer } from 'ember-cli-fastboot-testing/test-support';
import noteFactory from '../../mirage/factories/note';

module('FastBoot | notes list test', function(hooks) {
  setup(hooks);

  test('it has the correct page title', async function(assert) {
    let note = new noteFactory().build();
    await mockServer.get(hostname, '/api/note/1', { note });

    let { htmlDocument } = await visit('/notes/1');

    assert
      .dom('head title', htmlDocument)
      .hasText('A test note');
  });
});
```

**Comment by KalachevDev** [comment]
@ryanto sure! @CvX we do a bit different:
Created such helper: https://gist.github.com/KalachevDev/e192dc68769803da22a5412b7af602d2
And use it following way in tests:

`await mockServer.get(urlToPassToMirage).resolve()`

**Comment by ryanto** [comment]
Awesome, thanks!

This was released as version `0.0.17`

**Comment by KalachevDev** [comment]
Thanks, @ryanto
